core/src/components/select/select.scss

```diff
@@ -77,7 +79,7 @@ button {
 
   text-overflow: ellipsis;
 
-  white-space: nowrap;
+  white-space: inherit;
```

Comment:
With Ionic 7 we updated `ion-select` to contain both the native control and the label. Will this impact the label when we sync with the v7 branch?

Reply:
I tested it out on the v7 feature branch and it seems applying it in the same places does still wrap the label properly.

Reply:
One more question -- did you test this with the new select syntax?

```
<ion-select label="My Label">...</ion-select>
```

v7 is backwards compatible with the old select syntax, so you'll need to check using the new syntax too.

Reply:
Perfect! Then this change looks good to me 👍